lib/puppet_x/puppetlabs/powershell_manager.rb

```diff
@@ -43,6 +43,10 @@ def initialize(cmd, debug, pipe_timeout)
 
         named_pipe_name = "#{SecureRandom.uuid}PuppetPsHost"
 
+        if !ENV["lib"].nil? && !File.exist?(ENV["lib"])
```

Comment:
`lib` is a variable much like path which can be a delimited string of multiple directories. This check will need to split the string and iterate to check each path. If any of the paths are not valid the module will fail.

`directory?` should also probably be used here instead of `exist?`. If someone were to use a real file in this var, instead of a directory, then `exist?` would return true, but that would still be an invalid value for this variable.